contentlayer.config.ts

```diff
@@ -363,6 +378,31 @@ export const Transcript = defineDocumentType(() => ({
     source_file: { type: "string" },
   },
   computedFields: {
+    tagsDetailed: {
+      type: "list",
+      resolve: (doc) => {
+        const topicsMap = doc?.tags as any || [];
+        const topics = (topicsMap?._array as string[]) ?? [];
+
+        const topicsWithTitles = topics.map((topic) => {
+          const currentTopic = getTopics().find(
+            (topicData: ContentData) => topicData.slug === topic
```

Comment:
we are not accounting for aliases here, as some tags could fall under aliases in the topic index and not the general tags.

There really isn't an easier way to get that data, we can discuss it further in this pr cc: @kouloumos